docs/ops.html

```diff
@@ -3819,6 +3819,9 @@ <h3>Provisioning the KRaft controller quorum</h3>
 # ZooKeeper client configuration
 zookeeper.connect=localhost:2181
 
+# The inter broker listener in brokers to allow KRaft controller send RPCs to brokers
+inter.broker.listener.name=PLAINTEXT
```

Comment:
just setting this seems to be not enough, it should be added to the `listener.security.protocol.map` as well otherwise there is a Kafka validation which fails.

Reply:
Good point. PR updated.